src/main/java/edu/kis/powp/jobs2d/drivers/DriverUsageDecorator.java

```diff
@@ -0,0 +1,61 @@
+package edu.kis.powp.jobs2d.drivers;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.util.logging.Logger;
+
+public class DriverUsageDecorator implements Job2dDriver {
+
+    private final int PRINT_WAIT_TIME_MS = 300;
+    private boolean isPrintAvailable = true;
+    private int currentX = 0, currentY = 0;
+    private double headDistance = 0, opDistance = 0;
```

Comment:
What is the point of DriverUsageDecorator if we cannot use the monitored values (e.g., there are no public getters)